### DISCUSSION TITLE
Install problems

### DISCUSSION THREAD

**Discussion by dviljoen** [golangci/golangci-lint#3115] [Q&A] [answered]
When trying to install I get: 
`$ go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.47.3
../../contrib/go/pkg/mod/honnef.co/go/tools@v0.3.3/go/ir/builder.go:36:2: //go:build comment without // +build comment`

**Comment by ldez** [comment, marked as answer]
Hello,

I think you are using Go < 1.17.

Only the 2 latest Go minor releases are supported by the Go team, currently go1.18 and go1.19.

https://golangci-lint.run/usage/faq/#which-go-versions-are-supported

It's the same for golangci-lint.
